GitSearch/Models/PaddedLabel.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PaddedLabel.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/18/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import UIKit
+
+class PaddedLabel: UILabel {
```

Comment:
It make sense to add `UIEdgeInsets` property with default value, so that everybody can tailor this subclass for their needs.

Reply:
What about label?

Reply:
Label has different padding value, and 20pxl don't match needed 5pxls

Reply:
I meant what about adding `UIEdgeInsets` property to label as well (with its own, different default)?